lexicon/bpe.py

```diff
@@ -85,14 +100,27 @@ def _fill_vocab_and_lexicon(self):
                     vocab_file.write(symbol + " -1\n")
                     symbol = symbol.replace(".", "_")
                     vocab.add(symbol)
-                    lexicon.add_phoneme(symbol.replace(".", "_"))
+                    if self.add_all_bpe_phonemes:
+                        lexicon.add_phoneme(symbol.replace(".", "_"))
 
         return vocab, lexicon
 
+    def _fill_additional_words(self):
```

Comment:
```suggestion
    def _get_additional_words(self):
```
A function that returns the thing it is named after I would call "get" whereas "fill" implies that there is some existing container in the class that is filled inplace

EDIT: nevermind.. the above functions also already are called "fill" and returns stuff.. then better stay consistent.